### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: resources/views/backend/users/index.blade.php [hunk 1]
```diff
@@ -80,10 +64,9 @@
                                                     </div>
                                                 </div>
                                             </div>
-                                        
-                                        <td>{!! $value->email !!}</td>
+                                        <td>{!! str_limit($value->email,20) !!}</td>
                                         <td>{!! $value->phone !!}</td>
-                                        <td>{!! $value->address !!}</td>
+                                        <td>{!! str_limit($value->address,20) !!}</td>
```
Comment (BunnyPi04) [inline resources/views/backend/users/index.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

File: app/Http/Controllers/Employee/VacationController.php [hunk 2]
```diff
@@ -93,8 +92,8 @@ public function update(VacationUpdateRequest $request, $id)
     {
         $start = $request->date_start;
         $end = $request->date_end;
-        if ($start >= $end) {
-            return redirect()->back()->with('error',  __('update_fail'));;
+        if ($start <= \Carbon\Carbon::now()) {
+            return redirect()->back()->with('error', __('update_fail'));;
```
Comment (BunnyPi04) [inline app/Http/Controllers/Employee/VacationController.php]: bị 2 dấu ; kìa

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Pull request merged**